base/cvd/cuttlefish/host/commands/cvd/server_command/start.cpp

```diff
@@ -728,31 +732,21 @@ Result<cvd::Response> CvdStartCommandHandler::Handle(
       }
     }
   });
+  auto launch_res = LaunchDeviceInterruptible(std::move(command),
+                                              group_creation_info, request);
+  StopHandlingInterruptSignals();
+  interrupter_thread.join();
+  auto response = CF_EXPECT(std::move(launch_res));
 
-  // Interrupts must be custom handled until PostStartExecutionActions returns.
-  android::base::ScopeGuard guard([&interrupter_thread]() {
-    StopHandlingInterruptSignals();
-    interrupter_thread.join();
-  });
-  CF_EXPECT(UpdateInstanceDatabase(group_creation_info));
-
-  Command command =
-      CF_EXPECT(ConstructCvdNonHelpCommand(bin, group_creation_info, request));
-
-  ShowLaunchCommand(command.Executable(), group_creation_info);
-  CF_EXPECT(request.Message().command_request().wait_behavior() !=
-            cvd::WAIT_BEHAVIOR_START);
-  CF_EXPECT(subprocess_waiter_.Setup(command.Start()));
+  // Print new group spec
+  auto group = CF_EXPECT(instance_manager_.FindGroup(InstanceManager::Query(
+      selector::kGroupNameField, group_creation_info.group_name)));
+  auto group_json = CF_EXPECT(status_fetcher_.FetchGroupStatus(group, request));
+  auto serialized_json = group_json.toStyledString();
+  CF_EXPECT_EQ(WriteAll(request.Out(), serialized_json),
+               serialized_json.size());
```

Comment:
nit: without a server `request.Out()` is stdout, right? So this can be
```c++
std::cout << group_json.toStyledString();
```

Reply:
I think there are other uses the the stdio members of the request object in other commands. I prefer to keep all of those consistent and replace them as a whole in a later PR if needed.